image/templates/helm/stackrox-secured-cluster/values-public.yaml.example

```diff
@@ -250,6 +250,17 @@
 #               matchLabels:
 #                 app: admission-control
 #
+#  # If admission controller webhook need specific namespaceSelector, you can specify the corresponding selector here.
```

Comment:
Hi, sorry for the late reply, I opened here https://github.com/openshift/openshift-docs/pull/84525 not sure if something is missing since no preview is available

Reply:
I posted a comment there and also triggered the preview pipelines.

Reply:
BTW the above comment is still pending.

Reply:
done